### PR TITLE
Change URLs to prepare for GitHub rename

### PR DESCRIPTION
**Pull request by dcolligan**
Issue #1585

### CODE REVIEW

**Comment by dcolligan** [comment]
The only changes I am unsure about are the Docker instructions in `docs/installation.rst`

File: docs/installation.rst [hunk 1]
```diff
@@ -193,13 +193,13 @@ If you already have a dataset on your machine, you can download and deploy the a
...
 
-  docker run -e GA4GH_DATA_SOURCE=/data -v /my/ga4gh_data/:/data:ro -d -p 8000:80 --name ga4gh_server ga4gh/server:latest
+  docker run -e GA4GH_DATA_SOURCE=/data -v /my/ga4gh_data/:/data:ro -d -p 8000:80 --name ga4gh_server ga4gh/ga4gh-server:latest
 
 Replace ``/my/ga4gh_data/`` with the path to your data.
 
 This will:
 
-* pull the automatically built image from `Dockerhub <https://registry.hub.docker.com/u/ga4gh/server/>`_
+* pull the automatically built image from `Dockerhub <https://registry.hub.docker.com/u/ga4gh/ga4gh-server/>`_
```
Comment (david4096) [inline docs/installation.rst]: This might be managed separately? Does it automatically redirect from github or is this a managed URL?

Comment (dcolligan) [inline docs/installation.rst]: Can't say for certain, but it looks like it doesn't.  Do we know who is managing / created the docker image?

Comment (david4096) [inline docs/installation.rst]: @kozbo -- I see @macieksmuga's comment on there and I believe we did have to regain control of that account. It says it's a public automated build off the repo.

Comment (kozbo) [inline docs/installation.rst]: Yes the build is automated. I created #1592 #1593 to track that the current docker build is failing, and we need to update the docs on the docker site.

Comment (david4096) [inline docs/installation.rst]: I just tested today and doing the repository rename in github's interface did automatically set up a redirect.

**Comment by david4096** [comment]
I just tested today and doing the repository rename in github's interface did automatically set up a redirect.

**Comment by dcolligan** [comment]
Great, that's good to know.

https://github.com/blog/1508-repository-redirects-are-here

**Comment by david4096** [comment]
Woohoo congratulations we are now one of many possible servers not just `server`.

You can update your git remote like this:

`git remote set-url upstream git@github.com:ga4gh/ga4gh-server.git`

**Comment by kozbo** [comment]
Should we add a message a the top of our README to let developers know?

**Comment by david4096** [comment]
Git is very kind about giving you a warning, so probably not.

**Pull request merged**